File: JanperCab/3-Application/Dtos/Enquiry/EnquiryForRunSheetDto.cs
namespace _3_Application.Dtos.Enquiry
{
    public class EnquiryForRunSheetDto : EnquiryForSheetDto
    {
        public int? DeliveryRunSheetId { get; set; }

        public EnquiryForRunSheetDto(_1_Domain.Enquiry enquiry) : base(enquiry)
        {
            DeliveryRunSheetId = enquiry.DeliveryRunSheetId;
        }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/enquiry/EnquiryForRunSheetDto.ts
import { EnquiryForSheetDto } from './EnquiryForSheetDto';

export class EnquiryForRunSheetDto extends EnquiryForSheetDto {
  deliveryRunSheetId: number;

  constructor() {
    super();
    this.$type =
      '_3_Application.Dtos.Enquiry.EnquiryForRunSheetDto, 3-Application';
  }

  getFullAddress = (): string => {
    return `${this.address}, ${this.suburb} ${this.state} ${this.postcode}`;
  };
}
